### ISSUE TITLE
No pop-up message when taking a screenshot

### ISSUE THREAD

**Issue by LB-KacperMetelski** [o3de/o3de#14722]
**Describe the bug**
When taking a screenshot, there is no output in the console or a pop-up message related to this action, despite the screenshot being taken.

This issue occurs on both Linux and Windows. However, on Windows there is a console window displayed for a split second.

**Steps to reproduce**
1. Open the Material/Script Canvas.
2. Capture a screenshot.

**Expected behavior**
A pop-up message informing about the screenshot is displayed.

**Actual behavior**
No indication related to the screenshot being captured.

**Screenshots/Video**
https://user-images.githubusercontent.com/110163473/220658575-1f49b231-9814-4b86-ab6a-774c1851f14c.mp4

**Found in Branch**
Development

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
9c57ede

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows
 - Version: 11 22H2
 - CPU: I7-11700f
 - GPU: Geforce RTX3070 MSI
 - Memory: 32GB

**Comment by monroegm** [comment]
Note from SIG-Content Triage: It doesn't necessarily need to be a pop up, but we need some sort of visual feedback. @o3de/sig-ui-ux please advise

**Comment by bhanuja-s** [comment]
I agree, I don't want to disrupt the user's task – we can simply show them a Toast. We have guidelines on O3DE Docs: https://www.o3de.org/docs/tools-ui/ux-patterns/error/components/toasts/ on how to use them; they are a more passive and non-disruptive manner of notifying user. If absolutely necessary, you can use a Toast with Link which is a persistent notification, and the user has to manually dismiss it – it also gives the user a chance to view their screenshot.

> Screenshot saved. View your screenshot here.
